src/bindings/python/wheel/requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
-setuptools>=53.0.0,<=65.7.0
+setuptools>=65.6.1,<=65.7.0
```

Comment:
1. is it Windows only issue? should we increase a version only for that platform?
2. is there any way to WA issue on OpenVINO side?

Reply:
do we know why do we have byte-compiling?
```
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_adapters.py to test_adapters.cpython-310.pyc
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_caffe_launcher.py to test_caffe_launcher.cpython-310.pyc
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_config_reader.py to test_config_reader.cpython-310.pyc
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_config_validator.py to test_config_validator.cpython-310.pyc
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_dataset.py to test_dataset.cpython-310.pyc
byte-compiling C:\a\1\_w\build\tools\openvino_dev\build\accuracy_checker\Lib\site-packages\tests\test_dependency.py to test_dependency.cpython-310.pyc
```
I suppose this is not required during wheel build.

Reply:
> is it Windows only issue? should we increase a version only for that platform?

In general we intend to avoid such practices as much as possible, because our requirements are complicated enough as it is (see ticket 98878). Also, `setuptools==65.6.1` has been released on November 23, 2022, which gives user some time window for upgrade.

In my opinion we should avoid OS-specific versions if possible. Also, upper bound can be deleted once https://github.com/openvinotoolkit/openvino/pull/16021 is merged.